src/index.html

```diff
@@ -1,12 +1,172 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+
+    <div class="container">
```

Comment:
Implement **start-day** modifier for the calendar with mon, tue, wed, thu, fri, sat and sun values (use @each)


Add a **modifier month-length** for the calendar with values 28, 29, 30 and 31 (use @for)